src/Tools/ILAsm/IlAsmDeploy.csproj

```diff
@@ -12,9 +12,6 @@
     <TargetLatestRuntimePatch>false</TargetLatestRuntimePatch>
     <SelfContained>true</SelfContained>
 
-    <!-- Do not build again when publishing -->
-    <NoBuild>true</NoBuild>
```

Comment:
This is unrelated to the fix, but allows for builds in VS to not fail. This same change is already in the master branch.